packages/vue3-flicking/src/Flicking.ts

```diff
@@ -75,23 +126,19 @@ class Flicking extends Vue.with(FlickingProps) {
     if (this.status) {
       flicking.setStatus(this.status);
     }
-  }
-
-  public beforeDestroy() {
+  },
+  beforeDestroy() {
```

Comment:
I think you can use beforeUnmount.

Reply:
Looks like I should, thanks for checking 👍